eng/performance/maui_scenarios_android.proj

```diff
@@ -25,19 +29,19 @@
   </ItemDefinitionGroup>
 
   <ItemGroup>
-    <MAUIAndroidScenario Include="Maui Android Default">
-      <ScenarioDirectoryName>mauiandroid</ScenarioDirectoryName>
-      <PayloadDirectory>$(ScenariosDir)%(ScenarioDirectoryName)</PayloadDirectory>
-      <ApkName>com.companyname.mauiandroiddefault-Signed</ApkName>
-      <PackageName>com.companyname.mauiandroiddefault</PackageName>
-    </MAUIAndroidScenario>
-    <MAUIAndroidScenario Include=".NET Android Default">
+    <MAUIAndroidScenario Include=".NET Android Default $(RunConfigsString)">
```

Comment:
I think we should revisit the naming here and/or introduce new cases.
The `Default` is different between Mono, CoreCLR and NativeAOT, Mono's defaults to profiled AOT, while CoreCLR defaults to JIT only *at the moment* etc, which can lead to confusion when analysing the measurements.
I suggest we add new more descriptive scenarios (JIT, AOT, ..) conditioned on the runtime type.
Lastly, marking which one of the scenarios is the default one is still valueable.

Reply:
Added

Reply:
Actually, I'm thinking about removing the `$(RunConfigsString)` all together from the scenario name as per 
https://github.com/dotnet/runtime/pull/114117#discussion_r2023567496 and offline discussion with @LoopedBard3 . 

We can already see the configuration used in the job name 
<img width="611" alt="image" src="https://github.com/user-attachments/assets/59cd5c4a-2864-4c7d-9130-7f9d453dd2b0" />
+ everything we place in `configurations` https://github.com/dotnet/performance/blob/2659b9d72210c725f20e7a10499ff88d40bcc29f/scripts/run_performance_job.py#L550-L551 is going to flow to database so we can retrieve it for visualizations...